api/middleware.go

```diff
@@ -99,10 +98,36 @@ func (api *API) loadInstanceConfig(w http.ResponseWriter, r *http.Request) (cont
 	return ctx, nil
 }
 
-func (api *API) verifyNetlifyRequest(w http.ResponseWriter, req *http.Request) (context.Context, error) {
-	token := strings.TrimPrefix(req.Header.Get("authorization"), "Bearer ")
-	if token != api.config.NetlifySecret {
-		return nil, unauthorizedError("Request did not originate from Netlify")
+func (api *API) verifyOperatorRequest(w http.ResponseWriter, req *http.Request) (context.Context, error) {
+	c, _, err := api.extractOperatorRequest(w, req)
+	return c, err
+}
+
+func (api *API) extractOperatorRequest(w http.ResponseWriter, req *http.Request) (context.Context, string, error) {
+	token, err := api.extractBearerToken(w, req)
+	if err != nil {
+		return nil, token, err
 	}
-	return req.Context(), nil
+	if token != api.config.OperatorToken {
```

Comment:
this is checking if the token matches, not just extracting it. It would be something like 

```
if token != "" {
....
}
```